README.md

```diff
@@ -1,31 +1,34 @@
 # nRF Connect Power Profiler
 
-Power Profiler app for
-[nRF Connect](https://github.com/NordicSemiconductor/pc-nrfconnect-launcher).
-
-## Introduction
-
-_nRF Connect Power Profiler_ is a tool to communicate with the
+nRF Connect Power Profiler is a tool to communicate with the
 [Power Profiler Kit II (PPK2)](https://www.nordicsemi.com/Software-and-tools/Development-Tools/Power-Profiler-Kit-2),
 an affordable and flexible tool to obtain real-time current measurements of your
-designs. It also supports the
-[older, first version of the Power Profiler Kit](https://www.nordicsemi.com/Software-and-tools/Development-Tools/Power-Profiler-Kit).
+designs.
+
 The PPK measures current consumption for a connected Nordic Development Kit or
 any external board. It gives a detailed picture of the current profile for the
 user application.
 
-All functionality is described in the
-[User Guide](https://infocenter.nordicsemi.com/topic/ug_ppk2/UG/ppk/PPK_user_guide_Intro.html)
-(If you use the first version of the hardware, there is a different
-[User Guide for the PPK 1](https://infocenter.nordicsemi.com/topic/ug_ppk/UG/ppk/PPK_user_guide_Intro.html)).
-
 ![screenshot](resources/screenshot.png)
 
+The application also supports the
+[older, first version of the Power Profiler Kit](https://www.nordicsemi.com/Software-and-tools/Development-Tools/Power-Profiler-Kit).
+
```

Comment:
PPK 1 is deprecated and not supported. Last app version that has know support for this is PPK 3.5.5 with nrf connection for desktop 4.4.0